doc/scenario.md

```diff
@@ -86,17 +89,23 @@ In addition to the default scenario (see [steps](#steps)), there are also the fo
   - [configure_instance](#configure_instance)
   - [restart_instance](#restart_instance)
   - [wait_instance_started](#wait_instance_started)
```

Comment:
You forget about variables for this task

Reply:
Removed them. I will modify that in separated PR.